### ISSUE TITLE
Another Panel that allows darker colors without added slider

### ISSUE THREAD

**Issue by SilasStokes** [alabsi91/reanimated-color-picker#23]
Hi - love this project. I am currently using it to work on a basic react app. I was wondering if there was any roadmap to expand and add more Panels? Ideally I am looking for one that gives Panel 3 level control but also allows the brightness slider to be embedded into the Panel such that the end user can get darker shades without needing to adjust another slider.

I'd be happy to give implementing a go if you're open to a pull request.

**Comment by alabsi91** [comment]
Hello, thank you for your feedback and interest in this project. I welcome pull requests and contributions from the community. If you are willing to implement the feature you suggested, that would be great. It would be helpful if you could provide an example of the desired panel appearance, either by referencing an existing picker or an image. I look forward to seeing your work.

**Comment by SilasStokes** [comment]
Thanks for the response. I was thinking something like the picture I've attached - this was taken from Freeform the iOS drawing app.  I could see this being a diamond shape where the black and white converge at opposing tips. I will take a look at the codebase and see if I can implement it. 

![IMG-0017](https://user-images.githubusercontent.com/63477128/228063237-b3090dba-4ae2-4015-9881-1ba0d293dbc4.jpg)

**Comment by alabsi91** [comment]
I now see what you're aiming for. This panel looks easy to use, but it has a drawback - you can't adjust the saturation and brightness at the same time. Therefore, you can't achieve certain colors such as gray, for example. 
However, I believe it's relatively easy to implement. Also, I appreciate the blurry effect, and I might consider adding a similar effect to other sliders as well. 

All the assets used in the color picker are available at this [Figma link](https://www.figma.com/file/1NAZsgrXejzzDsakZtQyuP/reanimated-color-picker-assets?node-id=0%3A1&t=CZzURph1MOPimwI2-1) if you are interested to take a look.
 
By the way, I would be more than happy to assist with the implementation of this new panel whenever you need help.

**Comment by SilasStokes** [comment]
Wow that was quick! Thank you!

**Comment by SilasStokes** [comment]
I was thinking about how to incorporate gray as well, seems like you could just at saturation gradient on the top or bottom that would allow a black to gray to white band be available. I played around with your figma link above to see if I could get it and I have it roughly. I attached a picture below and here's a link to the figma design. Literally my first time ever using figma sorry if the edits don't make any sense. 

Fima Link: https://www.figma.com/file/wMqOQXNMOJxoKiY0lQaGC7/reanimated-color-picker-assets-(Copy)?node-id=34%3A21&t=crkgZosxcjeXEhpR-1

![Panel4](https://user-images.githubusercontent.com/63477128/228574009-e46c067d-01fc-45bf-aa4f-ca08f4acacd8.png)

**Comment by alabsi91** [comment]
The problem is not just the 'gray' color. The problem with this panel is that you can only adjust either the brightness or the saturation at a time. For instance, you can't have 50% brightness with 50% saturation. That's okay if the use case requires a simple color pick. However, I tried adding both the saturation and brightness sliders alongside the panel to see how it would react. If I changed the color to a value that couldn't be represented, it would glitch. I'm still experimenting with it, and I hope you'll also try it and share your opinion.

**Comment by SilasStokes** [comment]
ahhh I suspected the problem was more nuanced than it seemed. Appreciate your patience with me. I actually ran into a corollary to this problem yesterday when using the circular panel without any sliders where I was init'ing with random color hex codes. If there was any initial color that needed a slider to be present to be represented, the thumb would be stuck on the initial color, irregardless of the position.

I have time this weekend to find a solution - I don't mean to be inventing work for you haha. For my use case at least it would be enough to just turn off the brightness and saturation ability on panel 4, reducing it to just a color picker. Which is exactly what I am using it for.
